venues/auai.org/UAI/2018/python/update-recommendations-metadata.py

```diff
@@ -0,0 +1,57 @@
+import openreview
```

Comment:
Why don't we have a setup-metadata.py script only?

It creates or update metadata notes using all the inputs:

-tpms scores files
-bids
-recommendations
....

Reply:
instead of having one script per type of score

Reply:
ok, good idea

Reply:
i think this is good, but i haven't had time to implement a single updating script yet. I'm going to close this in favor of this branch (https://github.com/iesl/openreview-scripts/pull/664) which contains the changes here.